dicoogle/src/main/resources/webapp/sass/modules/_management.scss

```diff
@@ -31,6 +31,7 @@
       color: $blue1;
       &:hover {
         background: $hover_color;
+        color: #ffffff;
```

Comment:
can we also use variable here and also in sequel? :)

Butt very nice piece of work! :)

Reply:
We can indeed. Last commits replace that case and other cases of button color with a variable :)